main.py

```diff
@@ -58,3 +58,7 @@ def handle_scheduled_scraping(event, context):
         logger.info(f'adding {len(releases.releases)} to {releases.serie_id}, {releases.serie_title}')
         page_marks_map[releases.serie_id].extend(releases.releases)
     page_marks_db.batch_put(all_page_marks)
+
+
+if __name__ == '__main__':
+    handle_scheduled_scraping(None, None)
```

Comment:
Has deletion of local_main file been  prorperly committed?

Reply:
the file has never been committed in the first place. It is only now being updated.
Add instruction on how to run it in local in README?

Reply:
added basid instructions